rmgpy/kinetics/arrhenius.pyx

```diff
@@ -423,10 +423,10 @@ cdef class PDepArrhenius(PDepKineticsModel):
     
     """
 
-    def __init__(self, pressures=None, arrhenius=None, highPlimit=None, Tmin=None, Tmax=None, Pmin=None, Pmax=None, comment=''):
+    def __init__(self, pressures=None, arrhenius=[], highPlimit=None, Tmin=None, Tmax=None, Pmin=None, Pmax=None, comment=''):
```

Comment:
This was an old commit from awhile ago (which I fixed), I am not sure how it got in here. I will remove this commit from the PR.